src/ledger/NetworkConfig.cpp

```diff
@@ -499,6 +499,11 @@ initialBucketListSizeWindow(Application& app)
 {
     ConfigSettingEntry entry(CONFIG_SETTING_BUCKETLIST_SIZE_WINDOW);
 
+    if (!app.getConfig().MODE_ENABLES_BUCKETLIST)
+    {
```

Comment:
I think we should initialize the sliding window with all ones. We have asserts later in the code that check that this vector's size is always `BUCKET_LIST_SIZE_WINDOW_SAMPLE_SIZE`. We also take the sum of all these values as the denominator for division, so it shouldn't be zero or garbage initialized.

Reply:
Good idea. Done